onnxruntime/core/graph/function.cc

```diff
@@ -457,6 +457,17 @@ FunctionImpl::FunctionImpl(const onnxruntime::Graph& graph,
     }
   }
 
+  for (const auto& constant_initializer : meta_def->constant_initializers) {
+    const ONNX_NAMESPACE::TensorProto* initializer = nullptr;
+    if (graph.GetInitializedTensor(constant_initializer, initializer)) {
+      // meta_def->constant_initializer could have duplicates so make sure we only add once
```

Comment:
Can we use Graph::GetConstantInitializer here and make sure the value returned is not a nullptr?

Reply:
I think it needs an ORT_ENFORCE to check it's not nullptr. Otherwise you're going to silently carry on and the new GraphProto will be missing a required initializer which will be very obtuse to debug.